### PR TITLE
UpdateAttention deficit hyperactivity disorder

### PR DESCRIPTION
**Pull request by HariMahi**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ Please read the [Contributing Guidelines](./contributing.md) before contributing
   - [Schizophrenia](#schizophrenia)
   - [Borderline Personality Disorder](#borderline-personality-disorder-bpd)
   - [Prosopagnosia](#prosopagnosia)
+  - [Attention deficit hyperactivity disorder](#attention-deficit-hyperactivity-disorder)
```
Comment (icyflame) [inline readme.md]: d, h and d should be in uppercase. Fix the hyperlink (see the example of DKE and check this before submitting for review)

File: readme.md [hunk 2]
```diff
@@ -431,6 +432,21 @@ context. There have been no treatment methods discovered as of now.
 
 [R: National Institute of Neurological Disorders and Stroke](https://www.ninds.nih.gov/Disorders/All-Disorders/Prosopagnosia-Information-Page) 
 
+#### [Attention deficit hyperactivity disorder](https://en.wikipedia.org/wiki/Attention_deficit_hyperactivity_disorder) [ADHD]
```
Comment (icyflame) [inline readme.md]: d, h and d in uppercase

File: readme.md [hunk 3]
```diff
@@ -431,6 +432,21 @@ context. There have been no treatment methods discovered as of now.
...
+
+> A chronic condition including attention difficulty, hyperactivity and impulsiveness
+
+Attention deficit hyperactivity disorder (ADHD) is a mental disorder of the 
+neurodevelopmental type. It is characterized by problems paying attention,
```
Comment (icyflame) [inline readme.md]: problems _in_ paying attention

**Review by icyflame — CHANGES_REQUESTED** [review]

File: readme.md [hunk 4]
```diff
@@ -36,6 +36,7 @@ Please read the [Contributing Guidelines](./contributing.md) before contributing
   - [Schizophrenia](#schizophrenia)
   - [Borderline Personality Disorder](#borderline-personality-disorder-bpd)
   - [Prosopagnosia](#prosopagnosia)
+  - [Attention Deficit Hyperactivity Disorder](#attention-deficit-hyperactivity-disorder)
```
Comment (icyflame) [inline readme.md]: This hyperlink is still incorrect

**Review by icyflame — CHANGES_REQUESTED** [review]

**Comment by HariMahi** [comment]
#### [Attention Deficit Hyperactivity Disorder](https://en.wikipedia.org/wiki/Attention_deficit_hyperactivity_disorder)  [ADHD]
 
I have made the changes as you said already.

**Comment by HariMahi** [comment]
I have updated the hyperlink

File: readme.md [hunk 5]
```diff
@@ -431,6 +432,21 @@ context. There have been no treatment methods discovered as of now.
...
+Attention deficit hyperactivity disorder (ADHD) is a mental disorder of the 
+neurodevelopmental type. It is characterized by problems in paying attention, 
+excessive activity, or difficulty controlling behavior which is not appropriate 
+for a person's age. The symptoms appear before a person is twelve years old, are 
+present for more than six months, and cause problems in at least two settings 
+(such as school, home, or recreational activities).
+
+[R: National Institute of Mental Health](https://www.nimh.nih.gov/health/topics/attention-deficit-hyperactivity-disorder-adhd/index.shtml)
+
+[R: medlineplus.gov](https://medlineplus.gov/attentiondeficithyperactivitydisorder.html)
```
Comment (icyflame) [inline readme.md]: Ref too short.

**Review by icyflame — CHANGES_REQUESTED** [review]

**Pull request closed without merging**